client/src/plugins/version-info/ReleaseInfo.less

```diff
@@ -1,9 +1,10 @@
 :local(.ReleaseInfo) {
   overflow: auto;
+  max-height: 500px;
```

Comment:
Release info max height shall be limited by the parent container (cf. Section#maxHeight).